docs/gradle-plugin.md

```diff
@@ -86,3 +86,7 @@ Configuration options
 
 -   `includeIntrospectionTypes` - Include the introspection types in the
     schema. Default false.
+
+-   `federationEnabled` - Enable GraphQL Federation. This is automatically
+    enabled if any Federation-related annotations are found in your application, otherwise the default is false.
```

Comment:
The gradle plugin doesn't automatically check for these, does it? (I don't see that anywhere)

Reply:
I followed the example of maven plugin. Let me try to do automatic detection of federation annotations.